vault/resource_pki_secret_backend_root_sign_intermediate.go

```diff
@@ -303,7 +303,14 @@ func setCAChain(d *schema.ResourceData, resp *api.Secret) error {
 	field := "ca_chain"
 	var caChain []string
 	if v, ok := resp.Data[field]; ok && v != nil {
-		caChain = v.([]string)
+		switch v := v.(type) {
```

Comment:
Wanted to confirm, was this always returned as `[]interface{}` or was it previously being returned as `[]string` and the behavior changed recently?